bitnami/rabbitmq/templates/NOTES.txt

```diff
@@ -4,8 +4,8 @@
 Credentials:
 
     echo "Username      : {{ .Values.auth.username }}"
-    echo "Password      : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ include "rabbitmq.fullname" . }} -o jsonpath="{.data.rabbitmq-password}" | base64 --decode)"
-    echo "ErLang Cookie : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ include "rabbitmq.fullname" . }} -o jsonpath="{.data.rabbitmq-erlang-cookie}" | base64 --decode)"
+    echo "Password      : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "rabbitmq.secretPasswordName" . }} -o jsonpath="{.data.rabbitmq-password}" | base64 --decode)"
+    echo "ErLang Cookie : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ template "rabbitmq.secretErlangName" . }} -o jsonpath="{.data.rabbitmq-erlang-cookie}" | base64 --decode)"
```

Comment:
```suggestion
    echo "Password      : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ include "rabbitmq.secretPasswordName" . }} -o jsonpath="{.data.rabbitmq-password}" | base64 --decode)"
    echo "ErLang Cookie : $(kubectl get secret --namespace {{ .Release.Namespace }} {{ include "rabbitmq.secretErlangName" . }} -o jsonpath="{.data.rabbitmq-erlang-cookie}" | base64 --decode)"
```

Reply:
@andresbono What's the difference between `include` and `template`? I did what was already done:
https://github.com/bitnami/charts/blob/f5f3ad2cb7737cfe6ef3a822c3e70313a5616799/bitnami/rabbitmq/templates/statefulset.yaml#L150

And this seems to work (I just tested it locally). :)

Reply:
@anton-johansson in this case, both functions are actually equivalent. I just suggested the change to be consistent with the rest of the file.

Both functions behave differently if you use template pipelines (which is not the case). `include` supports it and `template` doesn't:

https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-include-function

Reply:
Thanks for the explanation! I have now changed.